tests/integrations/raftstore/test_lease_read.rs

```diff
@@ -312,3 +312,85 @@ fn test_node_callback_when_destroyed() {
         resp
     );
 }
+
+#[test]
+fn test_read_index_when_transfer_leader() {
+    let mut cluster = new_node_cluster(0, 3);
+
+    // Increase the election tick to make this test case running reliably.
+    configure_for_lease_read(&mut cluster, Some(50), Some(10_000));
+    let max_lease = Duration::from_secs(2);
+    cluster.cfg.raft_store.raft_store_max_leader_lease = ReadableDuration(max_lease);
+
+    cluster.pd_client.disable_default_operator();
+    let r1 = cluster.run_conf_change();
```

Comment:
Why not just `cluster.run()`?

Reply:
election timeout is 10s, `cluster.run()` needs 10s to elect a leader.